nilearn/_utils/__init__.py

```diff
@@ -9,3 +9,5 @@
 from .cache_mixin import CacheMixin
 
 from .logger import _compose_err_msg
+
+from .exceptions import DimensionError
```

Comment:
Fixed.

On Tue, Jun 16, 2015 at 1:09 PM, Loïc Estève <notifications@github.com>
wrote:

> In nilearn/_utils/__init__.py
> <https://github.com/nilearn/nilearn/pull/607#discussion_r32510889>:
>
> > @@ -9,3 +9,5 @@
> >  from .cache_mixin import CacheMixin
> >
> >  from .logger import _compose_err_msg
> > +
> > +from .exceptions import DimensionError
>
> It would make more sense to always import DimensionError from
> nilearn.exceptions rather than from nilearn._utils. You'll need to change
> the imports in the tests accordingly.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nilearn/nilearn/pull/607/files#r32510889>.
>